lib/components/footer/bottomNavigationTabs.dart

```diff
@@ -14,10 +14,10 @@ class BottomNavigationTabs extends StatelessWidget {
       child: Scaffold(
         backgroundColor: Colors.deepPurple,
         body: TabBarView(
-          physics: NeverScrollableScrollPhysics(),
+          physics: const NeverScrollableScrollPhysics(),
           children: <Widget>[
             homePage,
-            RecordingPage(),
+            const RecordingPage(),
             MySettingsPage(),
```

Comment:
why do you make RecordingPage() a const but not MySettingsPage() ?

Reply:
It's because MySettingsPage() does not have a constructor implemented. If, or when, it has a constructor, Flutter recommends making both the constructor and the class call const

Reply:
Makes sense!